report-service/src/main/java/gov/cdc/dataingestion/nbs/converters/Hl7ToRhapsodysXmlConverter.java

```diff
@@ -1,6 +1,5 @@
 package gov.cdc.dataingestion.nbs.converters;
 
```

Comment:
This file is too big and lots of if and else conditions. Is there any better way to make it simple